internal/templates/_partials.html

```diff
@@ -94,16 +94,16 @@
       <div>
         {{ if $.User.Is $.Twt.Twter.URL }}
           <a href="{{ $.User.URL | trimSuffix "/twtxt.txt" }}" class="u-url">
-            <img class="avatar u-photo" src="/user/{{ $.User.Username }}/avatar" />
+            <img class="avatar u-photo" src="/user/{{ $.User.Username }}/avatar" alt="" />
```

Comment:
```suggestion
            <img class="avatar u-photo" src="/user/{{ $.User.Username }}/avatar" alt="Avatar of {{ $.User.Username }}" />
```

Perhaps this is good enough?